ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PeerTable.java

```diff
@@ -51,6 +54,10 @@ public class PeerTable {
   private final Map<Bytes, Integer> distanceCache;
   private BloomFilter<Bytes> idBloom;
   private int evictionCnt = 0;
+  private final LHMWithMaxSize<String, Integer> ipAddressCheckMap =
```

Comment:
I think we should rename `LHMWithMaxSize` as `LinkedHashMapWithMaxSize` to be clearer.

Reply:
Will do :-)